appinventor/components/src/com/google/appinventor/components/runtime/Chart.java

```diff
@@ -221,6 +221,42 @@ public void Type(ChartType type) {
     }
   }
 
+  /**
```

Comment:
Since it looks like the code was just moved from line 575 to 224, it seems like we may just want to revert the changes to this file.